pkg/parser/parser.y

```diff
@@ -10265,8 +10265,8 @@ SetOprStmtWoutLimitOrderBy:
 			setOprList2 = []ast.Node{x}
 			with2 = x.With
 		case *ast.SetOprStmt:
-		    // child setOprStmt's limit and order should also make sense
-		    // we should separate it out from other normal SetOprSelectList.
+			// child setOprStmt's limit and order should also make sense
+			// we should separate it out from other normal SetOprSelectList.
```

Comment:
My editor fixed this based on `.editorconfig`.
https://github.com/pingcap/tidb/blob/master/pkg/parser/.editorconfig#L10

Tab should be used while spaces were used.
<img width="1015" alt="Screenshot 2023-12-28 at 17 10 13" src="https://github.com/pingcap/tidb/assets/32009080/0b948bc0-78f6-4a28-9854-633613d70763">